src/lib/components/streams/streams-manager.svelte

```diff
@@ -112,9 +120,16 @@
     } as TableData
 
     onMount(async () => {
+        // Get the current filter state from store and set value accordingly
+        const { authorFilterState } = get(channelFilterOptions)
+        streamsFilter[0].value = authorFilterState ? get(authenticatedUserDID) : undefined
```

Comment:
this line is no more necessary if you add the line above

Reply:
Line removed